### PR TITLE
JSM: Added module and TS files for SelectionBox and SelectionHelper.

### PR DESCRIPTION
**Pull request by Mugen87**

### CODE REVIEW

File: examples/jsm/interactive/SelectionBox.d.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export class SelectionBox  {
+  constructor(camera: Camera, scene: Scene, deep?: number);
+  camera: Camera;
+  collection: Mesh[];
+  deep: number;
+  endPoint: Vector3;
+  scene: Scene;
+  startPoint: Vector3;
+
+  select(startPoint: Vector3, endPoint: Vector3): Mesh[];
```
Comment (xawill) [inline examples/jsm/interactive/SelectionBox.d.ts]: @Mugen87 startPoint and endPoint should be optional.

Comment (Mugen87) [reply examples/jsm/interactive/SelectionBox.d.ts]: Oh, you are right! Do you want to make an PR with the change?

**Comment by mrdoob** [comment]
Late thanks! 😇

File: examples/jsm/interactive/SelectionBox.d.ts (hunk 1 above)
Comment (xawill) [reply examples/jsm/interactive/SelectionBox.d.ts]: Here it is #17689. Sorry for the delay

**Pull request merged**